src/test/java/seedu/address/view/ViewPartTest.java

```diff
@@ -13,7 +13,7 @@
 import javafx.fxml.FXML;
 import seedu.address.MainApp;
 
-public class UiPartTest {
+public class ViewPartTest {
 
     private static final String MISSING_FILE_PATH = "UiPartTest/missingFile.fxml";
```

Comment:
There are four lines here that are not updated yet (string content is still `Ui...`)